.github/workflows/kfp-kubernetes-test-python38.yaml

```diff
@@ -0,0 +1,28 @@
+name: kfp kubernetes tests with Python 3.8
+
+on:
+  push:
+    branches: [master]
+
+  pull_request:
+    paths:
+      - '.github/workflows/kfp-kubernetes-test-python38.yaml'
+      - 'sdk/python/**'
+      - 'api/v2alpha1/**'
+      - 'kubernetes_platform/**'
+
+jobs:
+  kfp-kubernetes-test-python38:
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.8'
```

Comment:
We need this test for Python versions 3.8 to 3.12. You can use a matrix here to loop over these versions. Example:

https://github.com/kubeflow/pipelines/pull/10926/commits/1737ef511431ebdc795e3b9315f03cecde44f61b#diff-e842340381d5ec7517ff5c2522b081e6b081cad5701ce9f06af7d088428974eeR18

Reply:
Done